src/ontology/uberon-edit.obo

```diff
@@ -2164,7 +2164,7 @@ xref: SCTID:118966000
 xref: UMLS:C1519343 {source="ncithesaurus:Skeletal_System_Part"}
 is_a: UBERON:0011216 {source="cjm"} ! organ system subdivision
 relationship: composed_primarily_of UBERON:0004765 ! skeletal element
-relationship: only_in_taxon NCBITaxon:7711 ! Chordata
+relationship: only_in_taxon NCBITaxon:7742 ! Vertebrata <vertebrates>
```

Comment:
When I suggested adding an explicit taxon restriction to vertebrates, I meant adding it to the new _paraspinal region_ term. No reason to change the existing taxon restriction on [subvidision of skeletal system](http://purl.obolibrary.org/obo/UBERON_0000075).

Reply:
@gouttegd yes, sorry, I will correct it now